src/kaleidoscope/driver/led/ws2812/config.h

```diff
@@ -43,6 +43,11 @@
 #define w_fixedhigh   4
 #define w_fixedtotal  8
 
+/* define CPU frequency in Mhz here if not defined in Makefile
```

Comment:
This should get pulled to its own commit.

Reply:
Yikes!  I think it should be removed, actually.  I put in in there so that some file(s) would get successfully processed by IWYU, with the intention of it being temporary.  Sorry!